tests/installation/bootloader_s390.pm

```diff
@@ -194,7 +194,7 @@ EO_frickin_boot_parms
 
     my $output_delim
       = $display_type eq "SSH" || $display_type eq "SSH-X" ? qr/\Q***  run 'yast.ssh' to start the installation  ***\E/
-      : $display_type eq "VNC" ? qr/\Q*** Starting YaST2 ***\E/
+      : $display_type eq "VNC" ? qr/\Q*** Starting YaST ***\E/
```

Comment:
Base VR https://openqa.nue.suse.com/tests/9721617#step/bootloader_s390/31 show this ONLY replace s/YaST2/YaST/ can not work here

Reply:
Need further check whether the adjust the trigger text work for non-SLES-15SP5.

Reply:
Currently seems blocked by issue [bsc#1203728](https://bugzilla.suse.com/show_bug.cgi?id=1203728)

Detail job link:
https://openqa.nue.suse.com/tests/9678576#step/bootloader_s390/31

Reply:
@skliu i suggest remove this change currently.

Reply:
the bug is unrelated, that message was there before, to boot the system we need to fix this.
we cannot change to 'YAST' I think because we will not be able to boot other products, please provide clear verification for current product and other products.

Reply:
Also needs to be fixed here:
`Test died: yast didn't start at sle/tests/installation/bootloader_zkvm.pm line 88.`

Reply:
> 
all seems passed.

https://openqa.suse.de/t9725225
https://openqa.nue.suse.com/t9725226
https://openqa.nue.suse.com/t9725227
https://openqa.nue.suse.com/t9725228
https://openqa.nue.suse.com/t9725229